helpers/codegen/codegen.js

```diff
@@ -42,6 +43,7 @@ export const codegens = [
   SalesforceApexCodegen,
   ShellHTTPie,
   WgetCodegen,
+  JavaOkHttpClientCodegen,
```

Comment:
move Java OK HTTP Client to `JSXHRCodegen`

Reply:
sure @liyasthomas

Reply:
raising PR again after the changes

Reply:
@liyasthomas pushed latest changed kindly merge PR now